src/tests/ftest/util/apricot/apricot/test.py

```diff
@@ -867,3 +867,32 @@ def add_container(self, pool, namespace=None, create=True):
                 to True.
         """
         self.container = self.get_container(pool, namespace, create)
+
+    def start_additional_servers(self, additional_servers):
```

Comment:
Maybe you can pass in the `index` as an optional argument (defaulted to 0) to the function to let the user specify which server_manager they would like to use?

Reply:
Done